src/canvas.class.js

```diff
@@ -1196,13 +1199,14 @@
     /**
      * @private
      */
-    _checkTarget: function(pointer, obj) {
+    _checkTarget: function(pointer, obj, globalPointer) {
+      globalPointer = globalPointer || pointer;
```

Comment:
is this check necessary? it looks like that we always pass in globalPointer.

Reply:
I wonder if someone might call this private method directly.
Should I remove this check or just document this method?

Reply:
Document the method is enough, is a private method.
If someone call it directly is gonna have problems anyway since those methods get changed with no information to devs.
We can put this check in searchPossibleTargets that while being private too, is at least one level up of this.

what do you think?

Reply:
That make sense to me.